v5/v5_with_library/v5_stable.ino

```diff
@@ -634,13 +650,30 @@ String getResponse() {
         }
       }
     } else {
-      Println(3, "skipping message from : " + senderNumber);
+      Println(3, "skipping message from: " + senderNumber);
       deleteMessage(newMessageNumber);
     }
   } else if (response.indexOf("+CLIP:") != -1) { // miss call
     //+CLIP: "03354888420",161,"",0,"",0
-    String temp_str = "Missed call from : " + fetchDetails(response, "\"", 1);
+    hangUp();
+    delay(500);
+    String callerNumber = fetchDetails(response, "\"", 1);
+    String temp_str = "Missed call from: " + callerNumber;
+    if (callerNumber.length() > 10 &&
+        (callerNumber.startsWith("+92") || callerNumber.startsWith("03"))) {
+      delay(200);
+      sendSMS("AOA, this is " + OWNER_NAME +
+                  "'s secondary number. This message is auto-generated, as I "
+                  "am currently unable to take calls on this device. For "
+                  "urgent matters, please contact me at " +
+                  MY_NUMBER + ". Thank you for your understanding.",
+              callerNumber);
+      temp_str += " asked him to contact you.";
+    } else {
+      temp_str += " unable to ask him to contact you!";
+    }
```

Comment:
Have you tested this functionality? did it work as expected in #152? I think it might send double message but not sure.

Reply:
Yes, I do, I ask my fired to call me and he receives the message to connect my primary number.